roles/nova-common/tasks/main.yml

```diff
@@ -14,11 +14,17 @@
 - file: dest=/var/log/nova state=directory mode=0755 owner=nova group=nova
 
 - name: nova config
-  action: template src={{ item }} dest=/etc/nova mode=0644
+  action: template src={{ item }} dest=/etc/nova mode=0640
+          owner=nova group=nova
   with_fileglob: ../templates/etc/nova/*
   notify:
     - restart nova services
 
+- name: set nova log file permissions
+  file: dest="{{ item.paths[0] }}" mode=0640 state=touch
+        owner=nova group=nova
+  with_items: nova.logs
+
```

Comment:
FYI, this will fail if there is ever more than one log file listed in a path (as is supported). May work, but could easily lead to unexpected results in the future.